test/validation/api/traffic_mngr/traffic_mngr.c

```diff
@@ -3836,11 +3847,20 @@ static void traffic_mngr_test_tm_create(void)
 
 static void traffic_mngr_test_shaper(void)
 {
-	CU_ASSERT(test_shaper_bw("bw1",   "node_1_1_1", 0, 1   * MBPS) == 0);
-	CU_ASSERT(test_shaper_bw("bw4",   "node_1_1_1", 1, 4   * MBPS) == 0);
-	CU_ASSERT(test_shaper_bw("bw10",  "node_1_1_1", 2, 10  * MBPS) == 0);
-	CU_ASSERT(test_shaper_bw("bw40",  "node_1_1_1", 3, 40  * MBPS) == 0);
-	CU_ASSERT(test_shaper_bw("bw100", "node_1_1_2", 0, 100 * MBPS) == 0);
+	int ret;
+
+	ret = test_shaper_bw("bw1",   "node_1_1_1", 0, 1   * MBPS);
+	CU_ASSERT(odp_cunit_ret(ret) == 0);
```

Comment:
This will become undecipherable. I'd rather use smth like `CU_ASSERT(!odp_cunit_ret(test_shaper_bw(....))`.

Reply:
One line of function wrapped under function wrapped under define.  And it likely will not fit to one line. I think current code is more simple to read.

Reply:
agree, will update.